functests/4_latency/latency.go

```diff
@@ -167,6 +167,78 @@ var _ = Describe("[performance] Latency Test", func() {
 			}
 		})
 	})
+
+	Context("with the cyclictest image", func() {
+		BeforeEach(func() {
+			testName = "cyclictest"
+			initMaximumLatencyValue(testName)
+
+			if profile.Spec.CPU.Isolated == nil {
+				Skip(fmt.Sprintf("Skip the cyclictest test, the profile %q does not have isolated CPUs", profile.Name))
+			}
+		})
+
+		It("should succeed", func() {
+			latencyTestPod = getLatencyTestPod(profile, workerRTNode, testName, []string{})
+			createLatencyTestPod(latencyTestPod)
+
+			// verify the maximum latency only when it requested, because this value can be very different
+			// on different systems
+			if maximumLatency == -1 {
```

Comment:
better log a message (or Skip) here

Reply:
Will add a log message for that, but what do you mean by skip? isn't ```return``` act as a skip in this context?

Reply:
surely with return the test won't go any further (thus won't fail) but will this be notified or logged in any way? at the end of the standard ginkgo report we have the count of test succeeded, failed and skipped.
We should make sure is trivial to understand what happened, adding an explicit skip seems the best approach, logging and returning is the second best, doing nothing silently is the less convenient option.

Reply:
I see. so I'm not sure that skip is correct, because even if we are not evaluating the latency values, we still made sure the binaries and mechanism are functions properly - which is the main purpose of the test though.

Reply:
In general, my ideas was:
1. Running test under the openshift CI with -1 to check if binaries work
2. Running test under the baremetal CI with 20us to see if have expected value for the latency